tests/Demo.py

```diff
@@ -120,7 +120,7 @@ def LocalMain(config) :
 
             # Polling for the "WorkOrderGetResult" and break when you get the result
             while("WorkOrderGetResult" in input_json_str1 and "result" not in response):
-                if response["error"]["code"] == 9:
+                if response["error"]["code"] != 5:
```

Comment:
If you don't mind, could you please use a constant variable or add a comment for the meaning of the value 5?

Reply:
This probably means pending.  I would just define a constant at the top of this file until we have a file with all the response codes defined.  Something like:
`PENDING_RESPONSE = 5`

Reply:
I will make use of WorkOrderStatus enum.